plugins/modules/swim_workflow_manager.py

```diff
@@ -1096,24 +1303,35 @@ def get_diff_import(self):
                 return self
 
             if self.want.get("import_type") == "remote":
-                import_payload_dict = {}
-                temp_payload = self.want.get("url_import_details").get("payload")[0]
-                keys_to_change = list(import_key_mapping.keys())
+                import_image_payload = []
+                temp_payloads = self.want.get("url_import_details").get("payload")
 
-                for key, val in temp_payload.items():
-                    if key in keys_to_change:
-                        api_key_name = import_key_mapping[key]
-                        import_payload_dict[api_key_name] = val
+                for temp_payload in temp_payloads:
+                    source_urls = temp_payload.get('source_url', [])
+
+                    for url in source_urls:
+                        import_payload_dict = {}
+
+                        if 'source_url' in import_key_mapping:
+                            import_payload_dict['sourceURL'] = url
```

Comment:
Don't we need to append the url in the 'SourceURL' as I believe we are importing list of images.

Reply:
its a string as we discussed in the call